js/main.js

```diff
@@ -1,2 +1,83 @@
 $(function () {
+    // Bot randomly selects rock paper scissors
+    const botChoices = ["Paper", "Scissors", "Rock"]
+
+
+    //User clicks on button
+    let userChoice
+    let botChoice
+    $("#rock").click(() =>{
```

Comment:
@Tambini nice set up of your 3 click events, extracting the logic used to generate the random choice (`botChoices[Math.floor(Math.random()*3)]`) to would remove duplication of having the run the same code 3 different times